config/env_sample.hjson

```diff
@@ -59,7 +59,7 @@
     "CSRF_COOKIE_SAMESITE": "None",
     # Any logs that django outputs, turn these off for production
     "DJANGO_LOG_LEVEL": "DEBUG",
-    "DJANGO_DEBUG": true,
+    "DJANGO_DEBUG": false,
```

Comment:
We should add more comments around specific to these settings. Maybe like

```
# This enables debugging of Django code and should only be enabled in development. 
# Setting this to true in production may reveal internal information such as settings to a user if they encounter an error.
"DJANGO_DEBUG": false,
```

We probably should also change `DJANGO_LOG_LEVEL` to `INFO` in the line above and also set DJANGO_TEMPLATE_DEBUG to false. 
```
# Setting this to true can help with debugging Django templates. This should be false in production and enabled if needed in development.
  "DJANGO_TEMPLATE_DEBUG": true,
````

Reply:
@jonespm In order to simplify the settings and avoid confusion, I have removed the variables "DJANGO_TEMPLATE_DEBUG" and "DJANGO_LOG_LEVEL", so only "DJANGO_DEBUG" controls the log levels. Please take another look.

Reply:
I agree tying `DJANGO_TEMPLATE_DEBUG `and `DJANGO_DEBUG `together makes sense.

I'd probably leave the log level independently configured but using this as the fallback makes sense.